extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -437,6 +438,20 @@ public String getUploaderAvatarUrl() throws ParsingException {
         return fixThumbnailUrl(url);
     }
 
+    @Override
+    public long getUploaderSubscriberCount() throws ParsingException {
+        final JsonObject videoOwnerRenderer = JsonUtils.getObject(videoSecondaryInfoRenderer, "owner.videoOwnerRenderer");
+        if (videoOwnerRenderer.has("subscriberCountText")) {
+            try {
+                return Utils.mixedNumberWordToLong(getTextFromObject(videoOwnerRenderer.getObject("subscriberCountText")));
+            } catch (final NumberFormatException e) {
+                throw new ParsingException("Could not get subscriber count", e);
+            }
+        } else {
+            return ITEM_COUNT_UNKNOWN;
```

Comment:
this should be either -1 or have its own constant.
I'd suggest to add an own constant, because that's the way we went with in #663

Reply:
Yes.

Reply:
Done, I haven't added any documentation for the variable though.